deployer/Dockerfile

```diff
@@ -10,12 +10,22 @@ ENV BIN_DIR=/opt/deploy \
     CREATE_KUBECONFIG=1 \
     JAVA_HOME=/usr/lib/jvm/jre-1.8.0-openjdk
 
-RUN yum install -y java-1.8.0-openjdk openssl && yum clean all
+RUN yum install -y java-1.8.0-openjdk-devel openssl groovy maven \
+    && yum clean all
 
 RUN mkdir -p ${HOME} ${BIN_DIR} && chmod 777 ${HOME} && chmod 555 ${BIN_DIR}
 COPY .  ${BIN_DIR}
 WORKDIR ${BIN_DIR}
 
 VOLUME ${HOME}
 ENTRYPOINT ["./run.sh"]
+
+# XXX openshift-restclient-java patch
+RUN git clone https://github.com/openshift/openshift-restclient-java.git \
+    && cd openshift-restclient-java \
+    && sed -i '453a			typeMappings.put("Node", k8eEndpoint);' \
```

Comment:
We can make a PR against the client so you dont have to do this and just pull in a specific version of the restclient